runtime/makelib/targets.mk.aix.inc.ftl

```diff
@@ -98,13 +102,22 @@ else
 endif
 
 ifeq (,$(findstring xlclang,$(notdir $(CC))))
-  # xlc options
-  CFLAGS += -q mbcs -qinfo=pro
+  ifeq ($(OMR_ENV_OPENXL),1)
+    # for openxl do nothing
+  else
+    # xlc options
+    CFLAGS += -q mbcs -qinfo=pro
+  endif
 else
- # xlclang options
+  # xlclang options
   CFLAGS += -qxlcompatmacros
 endif
-CFLAGS += -qlanglvl=extended -qarch=ppc -qalias=noansi -qxflag=LTOL:LTOL0 -qsuppress=1506-1108 -qstackprotect
+ifeq ($(OMR_ENV_OPENXL),1)
+  #openxl options
+  CFLAGS += -std=c89 -qarch=ppc -fno-strict-aliasing -fstack-protector
+else
+  CFLAGS += -qlanglvl=extended -qarch=ppc -qalias=noansi -qxflag=LTOL:LTOL0 -qsuppress=1506-1108 -qstackprotect
+endif
 CFLAGS += -D_XOPEN_SOURCE_EXTENDED=1 -D_ALL_SOURCE -DRS6000 -DAIXPPC -D_LARGE_FILES
```

Comment:
take notice maybe:  we need to revisit this flag setting in the future ... -qarch=ppc is pretty out of date and conservative.